kalite/updates/management/commands/languagepackdownload.py

```diff
@@ -112,6 +116,13 @@ def get_language_pack(lang_code, software_version, callback):
     logging.info("Retrieving language pack: %s" % lang_code)
     request_url = get_language_pack_url(lang_code, software_version)
     logging.debug("Downloading zip from %s" % request_url)
+
+    # aron: hack, download_file uses urllib.urlretrieve, which doesn't
+    # return a status code. So before we make the full request, we
+    # check first if the said lang pack url exists. If not, error out.
+    if requests.head(request_url).status_code == 404:
```

Comment:
@benjaoming the only important change here. The rest can be ignored as automatic linting by my editor.